src/course-outline/data/thunk.js

```diff
@@ -91,3 +92,16 @@ export function enableCourseHighlightsEmailsQuery(courseId) {
     }
   };
 }
+
```

Comment:
What if this request is executed without internet?
I think would be better to add handler for this case

Reply:
already working